src/types.ts

```diff
@@ -283,8 +293,6 @@ export interface CaptureOptions {
     $set?: Properties /** used with $identify */
     $set_once?: Properties /** used with $identify */
     _url?: string /** Used to override the desired endpoint for the captured event */
-    /** Some events are sent as passengers inside other events - e.g. heatmaps and web vitals, not all ingestion routes can process passengers */
```

Comment:
Are web vitals the only passenger event types now? Is it worth making them full events

Reply:
they aren't passengers... just a dangling comment